apps/calendar/style/ui.css

```diff
@@ -230,10 +230,6 @@ body[data-path="/day/"] #view-selector > .day > a:active {
   border-bottom: 0.1rem solid #C5C5C5;
 }
 
-#advanced-settings-view .account-list a {
-  -moz-padding-end: 4rem;
-}
-
```

Comment:
removed this since we don't have the arrows anymore and it was reducing the available area